src/modules/apps.js

```diff
@@ -236,7 +236,8 @@ export default {
       log.debug('Starting to uninstall app', app)
       const appRegex = new RegExp(`^${vendorPattern}\.${namePattern}@${wildVersionPattern}$`)
       if (!appRegex.test(app)) {
-        return log.error('Invalid app format, please use <vendor>.<name>@<version>')
+        log.error('Invalid app format, please use <vendor>.<name>@<version>')
```

Comment:
@firstdoit If the uninstall doesn't need the version anymore, we should update the message, no?